.travis.yml

```diff
@@ -9,7 +9,8 @@ env:
 - ARCH=i686
 
 go:
-- 1.8
+- 1.8.x
```

Comment:
Do we need to backward compatible?

Reply:
We do not have to be, but mc has been allowed to be built since pretty much 1.7